src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -6,6 +6,13 @@ export type StartIdentityVerificationGravityOutput = {
   identity_verification_flow_url: string
 }
 
+export type IdentityVerificationGravityResponse = {
```

Comment:
Wondering why this (and the other type above), are exported from this file - are they being used anywhere? This is a more general file for all authenticated loaders, feels a bit out of place.

#minor